core/lib/dal/migrations/20231213192041_snapshot-recovery.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE snapshot_recovery
+(
+    l1_batch_number        BIGINT    NOT NULL PRIMARY KEY,
+    l1_batch_root_hash     BYTEA     NOT NULL,
+    miniblock_number       BIGINT    NOT NULL,
+    miniblock_root_hash    BYTEA     NOT NULL,
+
+    is_finished            BOOL      NOT NULL,
```

Comment:
Just to highlight: As we've just had a conversation with @RomanBrodetski regarding (de)normalization, does this field still make sense? 🙃 AFAIU, at least in the initial iteration, `is_finished` will always equal `last_finished_chunk_id == total_chunk_count - 1`.

Reply:
corrected